libs/core/threading_base/include/hpx/threading_base/annotated_function.hpp

```diff
@@ -51,8 +51,9 @@ namespace hpx { namespace util {
     {
         HPX_NON_COPYABLE(annotate_function);
 
-        explicit annotate_function(char const* name)
-          : task_(thread_domain_, hpx::util::itt::string_handle(name))
+        explicit annotate_function(std::string const& name)
```

Comment:
I think a `std::string name` might be better here as it would allow avoiding double allocations.

Reply:
Good idea. Will fix that.